common/utils/string.ts

```diff
@@ -2,6 +2,6 @@ export function toHtmlId(s: string): string {
   return s.replace(/\W/g, '-');
 }
 
-export function removeHtmlTags(s: string): string {
-  return s.replace(/<[^>]+>?/gm, '');
+export function removeIdiomaticTextTags(s: string): string {
+  return s.replace(/<\/?[iI]+>/gm, '');
```

Comment:
We don't even need the `+`:

```suggestion
  return s.replace(/<\/?[iI]>/gm, '');
```